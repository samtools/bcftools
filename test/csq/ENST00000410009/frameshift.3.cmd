{bin}/bcftools csq -B 5 -f ENST00000410009.fa -g ENST00000410009.gff frameshift.vcf | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
