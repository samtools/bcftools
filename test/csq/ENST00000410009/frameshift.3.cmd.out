5507	G	GC	frameshift|CD207|ENST00000410009|protein_coding|-|25EPPPK..329>25GASSQ..94|5507G>GC,splice_region|CD207|ENST00000410009|protein_coding
